pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Used for reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM input file (-S) to BAM stream (-b) and save to file:
+
+`samtools view -bS {{input.sam}} > {{output.bam}}`
+
+- Take input from stdin (-) and output a region, including the file header (-h). Prints to stdout:
```

Comment:
To clarify, here it's ok to mention the link `stdin` --> `-`, because it's not obvious.